rest-java/src/main/java/org/hiero/mirror/restjava/config/NetworkProperties.java

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package org.hiero.mirror.restjava.config;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotEmpty;
+import java.util.List;
+import java.util.TreeSet;
+import lombok.Data;
+import lombok.Getter;
+import org.hiero.mirror.common.CommonProperties;
+import org.hiero.mirror.common.domain.entity.EntityId;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@ConfigurationProperties("hiero.mirror.rest-java.network")
+public class NetworkProperties {
+
+    @NotEmpty
+    private List<AccountRange> unreleasedSupplyAccounts = List.of(
+            new AccountRange(2, 2),
+            new AccountRange(42, 42),
+            new AccountRange(44, 71),
+            new AccountRange(73, 87),
+            new AccountRange(99, 100),
+            new AccountRange(200, 349),
+            new AccountRange(400, 750));
+
+    @Getter(lazy = true)
+    private final TreeSet<Long> unreleasedSupplyAccountIds = createUnreleasedSupplyAccountIds();
```

Comment:
Code smell here. Use `Set` or `Collection`. Whichever is highest level interface that works. Same for `createUnreleasedSupplyAccountIds()`.

```suggestion
    private final Set<Long> unreleasedSupplyAccountIds = createUnreleasedSupplyAccountIds();
```